### DISCUSSION TITLE
Number of labtests around 6 am and midnight has an unusual spike according to charttime of several biomarkers

### DISCUSSION THREAD

**Discussion by fikrynuhuman183** [MIT-LCP/mimic-code#1749] [MIMIC-IV]
During one of our analyses we observed that the number of labtests performed at 6 am to be high compared to other parts of the day, also we observed an unusual spike in the number of labtests at midnight (00:00). Here I am using the charttime as the labtest time. I am attaching one of the graphs we obtained below. It shows the number of lab tests performed at specific time intervals.
![image (17)](https://github.com/MIT-LCP/mimic-code/assets/100548085/541c9c1b-573d-4fbb-a7c3-4bdfc4334aff)

**Comment by alistairewj** [comment]
Morning labs. Routine. You could try filtering to only STAT labs to see if you still see the spike, I'd expect it to disappear.

**Reply by fikrynuhuman183** [reply]
Thank you, I will try that

**Reply by fikrynuhuman183** [reply]
Can you please let me know what you specifically mean by the STAT labs? I was using the labevents.csv to extract the information, how do we filter the 'STAT' labs from that data?

**Reply by alistairewj** [reply]
In MIMIC-IV there is a column called `priority` which is either ROUTINE or STAT. If you are using MIMIC-III it's not available.

**Comment by pszolovits** [comment]
Take a look at https://www.bmj.com/content/361/bmj.k1479

**Reply by fikrynuhuman183** [reply]
Thank you for the reply, I will take a look
